package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "private": true,
   "engines": {
-    "node": ">=6 <=8"
+    "node": ">=6 <=9"
```

Comment:
We should not commit this change, because that was just a "hack" to make your project work on your machine 😉